app/src/main/java/com/github/h3lp3rs/h3lp/database/FireDatabase.kt

```diff
@@ -24,11 +27,13 @@ internal class FireDatabase(path : String) : Database {
      * Only works on the following types (due to Firebase's policy):
      * - Boolean
      * - String
-     * - Int
+     * - Long

```

Comment:
Except that this is a lie.
Of course Firebase works with `Longs`, but here we adapted it such that it can only handle Int/String/Boolean (to satisfy **our** database requirements). Indeed, `Ints` are more used than `Longs` usually.